pkg/apiserver/statement/queries.go

```diff
@@ -212,14 +215,20 @@ func QueryPlans(
 	return
 }
 
-func QueryPlanDetail(
+func (s *Service) queryPlanDetail(
 	db *gorm.DB,
 	beginTime, endTime int,
 	schemaName, digest string,
-	plans []string) (result Model, err error) {
-	fields := getAllAggrFields()
+	plans []string,
+) (result Model, err error) {
+	tableColumns, err := s.params.SysSchema.GetTableColumnNames(db, statementsTable)
+	if err != nil {
+		return
```

Comment:
return nil, err

Reply:
Is that necessary? I mean we have named return values ```(result Model, err error)```. Or I need ```return Model{}, err```

Reply:
not so necessary and `result` is not a pointer 😀